apps/qubit/modules/informationobject/templates/_actions.php

```diff
@@ -70,4 +70,7 @@
 
   </ul>
 
+  <input type="button" id="fullwidth-treeview-reset-button" class="c-btn c-btn-submit" value="<?php echo __('Reset') ?>" />
+  <input type="button" id="fullwidth-treeview-more-button" class="c-btn c-btn-submit" data-label="<?php echo __('%1% more') ?>" value="" />
+  <span id="fullwidth-treeview-collection-url" data-collection-url="<?php echo sfConfig::get('app_siteBaseUrl') ?>/<?php echo $resource->getCollectionRoot()->getSlug() ?>"></span>
```

Comment:
Is there a reason to use the `siteBaseUrl` instead of the normal method to build the links in the templates?

Reply:
Hmm... I don't think there is. I'll make that change. ;)